### DISCUSSION TITLE
D3 graph into Flutter web

### DISCUSSION THREAD

**Discussion by MauroGentile** [d3/d3#3800] [Q&A] [answered]
I want to incorporate a D3 Graph into Flutter web, but I can't find a way.
Do you have any suggestions about this?

Thank you

**Comment by robifr** [comment, marked as answer]
late reply. i'm not a flutter dev, but i did with native android via webview. simple search from google, flutter supports webview too https://codelabs.developers.google.com/codelabs/flutter-webview#0
